src/enrichment.cc

```diff
@@ -55,7 +54,12 @@ void Enrichment::Build(cyclus::Agent* parent) {
   LOG(cyclus::LEV_DEBUG2, "EnrFac") << "Enrichment "
                                     << " entering the simuluation: ";
   LOG(cyclus::LEV_DEBUG2, "EnrFac") << str();
-  RecordPosition();
+  coordinates.RecordPosition(this);
```

Comment:
Didn't we record this in `EnterNotify`?  Which raises a tangential question - is there good documentation on the difference between `EnterNotify` and `Build`?

Reply:
It seems like someone put this in `Build` before because Enrichment didn't implement its own `EnterNotify`.  It should probably only be in one place, and I wonder if this feature is reason enough to implement `EnterNotify` or can `Build` be a substitute?